DeepSpeech.ipynb

```diff
@@ -172,7 +172,7 @@
     "export_version = 1\n",
     "remove_export = bool(int(os.environ.get('ds_remove_export', 0)))\n",
     "use_warpctc = bool(len(os.environ.get('ds_use_warpctc', '')))\n",
-    "default_stddev = float(os.environ.get('ds_default_stddev', 0.1))\n",
+    "default_stddev = float(os.environ.get('ds_default_stddev', 0.046875))\n",
```

Comment:
FWIW, TensorFlow has tf.contrib.layers.xavier_initializer which will adapt correctly to changes in number of inputs and outputs of the layers.